meetings/2023-04-19.md

```diff
@@ -0,0 +1,203 @@
+---
+parent: Meetings
+title: "2023-04-19"
+---
+
+# **ASWF TAC Meeting - 19 April, 2023**
+
+[Video Conference Link](https://zoom.us/j/97880950229?pwd=V3ZGM0RiK0dDSi9Ja0Z1ZitJZVB3dz09)
+
+## Voting member attendance
+
+### Premier member representatives
+
+* [ ] Bill Roberts - Adobe Inc.
+* [ ] Brian Cipriano - Google LLC / OpenCue
+* [x] Cory Omand - The Walt Disney Studios
+* [ ] Eric Enderton - NVIDIA / DPEL
+* [ ] Eric Reinecke - Netflix
+* [x] Esteban Papp - Amazon Web Services
+* [x] Gordon Bradley - Autodesk
+* [ ] Greg Denton - Microsoft
+* [x] Jean-Michel Dignard - Epic Games, Inc
+* [x] Kimball Thurston - Weta Digital Ltd / Chairperson / raw2aces
+* [x] Larry Gritz - Sony Pictures Imageworks
+* [ ] Mark Visser - Unity Technologies
+* [x] Matthew Low - DreamWorks Animation
+* [x] Michael B. Johnson - Apple Inc
+* [ ] Roy C Anthony - DNEG
+* [x] Sean C McDuffee - Intel Corporation
+* [ ] Sean O'Connell - Advanced Micro Devices (AMD)
+
+### Project representatives
+
+* [x] Carol Payne - OpenColorIO
+* [x] Cary Phillips - OpenEXR
+* [ ] Chris Kulla - Open Shading Language
```

Comment:
And I think I saw Chris on the call?

Reply:
Thanks for those, I swear there was a way to see the list of attendees for past meetings in the LFX console, but can't seem to find it right now. I still rely on taking a screenshot of the list of attendees at some arbitrary point in the meeting to make sure I didn't miss anyone at the end...